#Goal Speed Left & Right
float32 speedLeft
float32 speedRight
